lib/cocoapods/command/outdated.rb

```diff
@@ -22,10 +22,21 @@ def run
         if updates.empty?
           UI.puts 'No pod updates are available.'.yellow
         else
+          UI.section 'The color indicates what happens when you run `pod update`' do
+            UI.puts "#{'<green>'.green}\t\t - Will be updated to the newest version"
+            UI.puts "#{'<yellow>'.yellow}\t - Will be updated, but not to the newest version because of specified version in Podfile"
+            UI.puts "#{'<red>'.red}\t\t - Will not be updated because of specified version in Podfile"
+          end
```

Comment:
any chance you can add an extra newline here? Feels a bit cramped

Reply:
@orta sure, like this? 
<img width="656" alt="screen shot 2017-11-07 at 14 44 57" src="https://user-images.githubusercontent.com/3407787/32499834-f89d60c4-c3d3-11e7-9a79-012936c1c0da.png">

Reply:
Yep